CONTRIBUTING.md

```diff
@@ -1,79 +1,114 @@
 
-You should already be familiar with React 17 and or 18, JavaScript ES5/ES6/ES7.
+- **Documentation**: Possess strong documentation skills, and believe you can help elevate this projects' documentation even better? 
 
-## Style
+- **Code contribution**: Identify a bug in this app or thought of a new feature that could plunge this app to a new dimension? 
+
+## How to contribute
+
+Developers often derive more convenience in seeking permission from the project to contribute, so these next headings will get you started in no time contributing.
+
+## Select an issue
+
+If you are not sure what to work on, review the issues list first. There are also TODO's listed in the included [TODO.md](./TODO.md) file. 
+
+@gbowne1 and other project maintainers will assign users to issues on a first come, first serve basis. If you would like to work on an issue, feel free to indicate by tagging the mentioned persons.
+
+## Create an issue
+
+If you notice a typo, slight bug in the code and there are no lodged complaints on this yet, select the issues icon at the top, and on the right side of the next page select new issues and lodge your complain.
 
-I have not created an official style guide yet. I also have not created a Figma or layout. If anyone would like to create a style guide for us. Create a Discussion. I have typically let the community help
+## Fork this repository
 
-## Development
+After choosing what to work on, move to fork and setup your own repository.
 
-If you are not sure what to work on, review the issues list. There are also TODO's that will listed in the included [TODO.md](./TODO.md) file.
+On the project's repository page, look to the right side of the page to locate a fork tab, click on this icon, and this would redirect you to a new page with further instructions. 
 
-React gives us built in hooks such as useEffect, useSelector, useRef, useHook, useCallback, useContext, useMemo, useForm, useHistory, etc. Please make sure that we make the most use of what React gives us.
+![](./docs/fork.png)
+
+On this next page, if satisfied with the suggested project name click on create fork.
+
+![](./docs/CreateFork.png)
+
+Next,
+
+![](./docs/CreateForkBtn.png)
+
+This would automatically create your workspace/repository.
+
+![](./docs/ForkedBranch.png)
+
+## Editor & IDE
 
-The browser also gives us things like localStorage, IndexedDB, Session Storage, Cookies and Cache Storage. Please make use of them for
+The repository contains folders with project appropriate settings and configurations for Visual Studio code, but beyond that, we are tool and editor/IDE agnostic so you can use whatever editor or IDE or you like.
 
-Please also make sure you use accessibility (A11y) with themes and styles paying attention to focus styling, contrast and keyboard-accesibility.
+Follow the instructions in [DEVSETUP](/docs/DEVSETUP.md) to setup your IDE workspace.
 
-### PR's
+## Prerequisites to use Database for programmers
```

Comment:
This section doesn't appear to be using text for using the database.

The original heading I had ### PR's was intended to create short instructions on how to create and submit Pull Requests (PR's), this appears now at line 70

Reply:
I felt placing it after instructions on forking this repository and setting up the IDE was ideal, since they follow suit

Reply:
@Chizobaonorh I agree.